ironfish-cli/src/commands/status.ts

```diff
@@ -84,7 +84,9 @@ function renderStatus(content: GetNodeStatusResponse, debugOutput: boolean): str
   const avgTimeToAddBlock = content.blockSyncer.syncing.blockSpeed
 
   if (content.blockSyncer.status === 'syncing') {
-    blockSyncerStatusDetails.push(`${content.blockSyncer.syncing.speed} blocks per seconds`)
+    blockSyncerStatusDetails.push(
+      `${content.blockSyncer.syncing.speed} blocks added /s, ${content.blockSyncer.syncing.downloadSpeed} blocks downloaded /s`,
```

Comment:
I'm open to this being an "I'm dense" issue, but the /s isn't immediately obvious to me. I wonder if it might help to remove the space like `blocks added/s` or `blocks added/sec`? I don't feel too strongly about this

<img width="653" alt="image" src="https://user-images.githubusercontent.com/767083/195950159-acddbaf2-f20d-4619-8d50-40fc6a955a9f.png">

Reply:
yea I do like the `blocks added/sec`. I'll change to that